api/libproject/devstart.rb

```diff
@@ -1053,6 +1051,24 @@ def deploy_gcs_artifacts(cmd_name, args)
   :fn => ->(*args) { deploy_gcs_artifacts("deploy-gcs-artifacts", args) }
 })
 
+def deploy_gcs_demos(cmd_name, args)
+  ensure_docker cmd_name, args
+
+  common = Common.new
+  op = WbOptionsParser.new(cmd_name, args)
+  gcc = GcloudContextV2.new(op)
+  op.parse.validate
+  gcc.validate
+  common.run_inline %W{gsutil rm gs://#{gcc.project}-demos/**}
+  common.run_inline %W{gsutil cp demos/* gs://#{gcc.project}-demos/}
+end
+
+Common.register_command({
+  :invocation => "deploy-gcs-demos",
+  :description => "Deploys any GCS demos associated with this environment.",
+  :fn => ->(*args) { deploy_gcs_demos("deploy-gcs-demos", args) }
```

Comment:
Sounds fine to have this as a separate command, but wouldn't we want to push these automatically? Do we expect people to always manually run this in all environments any time we change a demo notebook?

Reply:
I think it is probably perfectly fine to push these automatically! Are they currently deployed as part of a circle workflow? (If so, have the circle files moved? I can't seem to find them)

I am not entirely certain what our plan is going to be for deployment long term though, as I think that is going to be a little bit played by ear... this is mostly temporary I think, to get the wiring set up to read demos from a google bucket. Eventually these are probably going to be stored and uploaded in/from a separate place, and we're just going to read them in.

Reply:
I would just make the deploy command call this method, I don't think it's worth explicitly including in the Circle configs.

Reply:
Should be good now!